### ISSUE TITLE
Split header bar in a similar way to several other elementaryos apps

### ISSUE THREAD

**Issue by jeremypw** [elementary/files#2456]
### Problem

The design of Files differs from other elementaryos apps that have a primary sidebar (Code, Music, Calendar etc)

### Proposal

Split the header and divide the header tools between them in a manner to be advised by the design team.

### Prior Art (Optional)

Gnome Files,  Elementary Code, Music, Calendar, SettingsPage

**Comment by danirabbit** [comment]
I had started playing around with mockups for redesigning the sidebar a while back:

![files-search-bigresults](https://github.com/user-attachments/assets/ea1de3bb-eaa6-48f0-80c6-636ec67136e8)

Looks like GNOME Files includs some kind of menu and a search of some kind in the sidebar header area:

![files-grid](https://github.com/user-attachments/assets/0a016336-3568-4b85-afeb-ba4dae259c00)

macOS Finder just includes window controls:

![macOS-Finder-1024x740-3949359649](https://github.com/user-attachments/assets/f83d7b19-45b2-4c91-83a6-7ec5d1d8045b)

iPadOS Files has a hide sidebar button and some kind of menu:

![iPad-Files-app-Column-view-799503786](https://github.com/user-attachments/assets/beffd689-cb77-4f3e-aff6-ad1ccda3ccdb)

**Comment by teamcons** [comment]
> I had started playing around with mockups for redesigning the sidebar a while back:
> 

This looks lovely ! Is it still being considered ?
Are there issues with this design ? Perhaps the file view buttons ? They may seem a bit disconnected from actual view if in sidebar

**Comment by jeremypw** [comment]
Cannot proceed with this until decided exactly how the tools are to be divided between the split headers.

**Comment by teamcons** [comment]
For the time being maybe the gtk4 port could keep the old design ?
so we could come up with a solution later on, this wouldnt be blocking

**Comment by jeremypw** [comment]
Yes, we try and separate porting from any design changes where possible.
